lib/appsignal/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Appsignal
-  VERSION = "4.2.0"
+  VERSION = "4.2.2"
```

Comment:
shall we not keep this in sync with the upstream? the version is anyways not relevant, since we're installing the gem from github